app/src/main/res/layout/activity_intro_3.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should be combined with activity_intro_1, so there are only two intro layouts